examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    validation_set_processed = wf.fit_transform(validation_set_dataset)
</pre><p>we dont use <code>fit</code> on validation set. it should be only <code>transform</code>.   so just this: <code>wf.transform(validation_set_dataset)</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>fixed</p>